tests/exporters/test_vhi.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+# from unittest.mock import patch, Mock
+import pytest
+
+from src.exporters.vhi import (
+    VHIExporter,
+    _parse_time_from_filename,
+    make_filename,
+    # download_file_from_ftp
+)
+
+import sys
+# get the root project directory
+project_dir = Path(__file__).resolve().parents[2]
+# append to sys
+sys.path.append(project_dir.as_posix())
+
+
+class TestVHIExporter:
+
+    def test_initialisation(self):
+        VHIExporter()
```

Comment:
What does this test?

Since the VHI initialization just does what the base exporter does, and that is tested in `test_base.py`, this test isn't necessary

Reply:
good point well made